### ISSUE TITLE
1.3.0c bytecode error, crash on mod init, teslapoweredthingies

### ISSUE THREAD

**Issue by olagarde** [davqvist/Exoria#360]
https://pastebin.com/ta4wCMmd

Exoria 1.3.0c crash during mod init, bytecode error callout for teslapoweredthingies, same for TPT 1.0.15.25, 1.0.14.17, 1.0.13.12, mod init passes with TPT removed, same results for SP and MP installs.

**Comment by davqvist** [comment]
That is odd, was it a fresh download? Maybe redownload the mod/the pack?

**Comment by olagarde** [comment]
Fresh download and install, also fresh local mod cache just in case.

- bush-master-core listed as TPT dependency but missing from install, will try adding
- forgelin at 1.6.0 in pack but 1.7.4 available, will try upgrading

**Comment by olagarde** [comment]
Same errors with latest ForgeLin and BushMasterCore (1.7.4 and 1.0.3 respectively).

**Comment by olagarde** [comment]
- Updated Java to latest 1.8, same thing.
- Reverted to previous 3 JRE in the 1.8 stack, same thing.
- Tried Java 1.10 compat STI for 1.8, same thing.
- Backtrace of all failures ends at bytecode validation failure for invalid goto, which can't be the problem if even one other user is loading the thing successfully (and many obviously are).
- No such failure with other mods that use bytecode pregeneration during mod init (similarly, there are many).
- Found reference to earlier JRE STI bug resulting in incorrect bytecode validation due to mixed javac resources, in turn due to multiple local JRE versions getting combined via improper JAVA_HOME var, but no such case here, I've both manually verified the calling path and checked the debug log trace for java resources and run the entire bloody thing in a virtualbox instance with a clean CENT7 install.

Any ideas, anyone?  I'm pretty much at a complete loss.  And yes, I've checked for other teslapoweredthingies users, no such complaint there.

**Comment by davqvist** [comment]
I have no idea either, have you made an issue at the TPT github and asked there?

**Comment by olagarde** [comment]
Yup.  As per advice there I'm trying a [very] old JRE.


null

**Comment by olagarde** [comment]
Fixed, sort of.  The fact that jdk versions outside 1.8 worked at all implies (requires, really) that there was a 1.8 dynlib orphaned ... somewhere.  Otherwise the static forge prereq  would fail, right?  So I manually restored the /Library/.../JavaVM.frameworks/ and /System/.../Java/ from prior to the issue, rebooted, upgraded to 1.8.181, rebooted, and it started working again.  So it probably was mixed versions, and definitely involved the base java loader dynlib (probably in /System/.../Java/ somewhere).  Beyond that, who knows?
